pkg/monitor/monitor.go

```diff
@@ -228,7 +186,7 @@ func (m *Monitor) logFailureDetails(ctx context.Context, retErr error) {
 		service = Service(srv)
 	}
 
-	m.Logger.Error("monitor job failure", serviceInstance, serviceDescriptor, service, zap.Error(retErr))
+	m.Logger.Error("monitor job failure", serviceDescriptor, service, zap.Error(retErr))
 }
 
 func (m *Monitor) serviceInstance(ctx context.Context, name, namespace string) (*v1beta1.ServiceInstance, error) {
```

Comment:
Remove this method if it's no longer being used

Reply:
done